Automattic/jetpack
# AI Assistant: Load Breve spelling dictionaries from CDN

Fixes #38887 

## Proposed changes:

* Loads the dictionary JSON from the backend, saving it to localStorage and reusing it on page refreshes

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Without sandboxing the public API, go to the block editor, with the Spelling Mistakes feature turned on
* Check that the dictionary fails to load, but nothing breaks
* Apply D158668-code and sandbox the public API
* Reload the block editor
* Check that the dictionary is loaded, with only one request to the endpoint. Note: The highlights are not properly re-loaded, but they appear if you click on a block. This will be tackled later.
* Reload the page
* Check that the dictionary is loaded from localStorage, without any requests
* Check that the typos are highlighted
